python/pyspark/ml/classification.py

```diff
@@ -656,6 +657,13 @@ class NaiveBayes(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol, H
                  HasRawPredictionCol):
     """
     Naive Bayes Classifiers.
+    It supports both Multinomial NB
```

Comment:
How about starting with a short version: "It supports both Multinomial and Binomial NB."
And then continuing with details: "Multinomial NB (link) can handle..."